backend/src/Controller/Image/ImageController.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace App\Controller\Image;
+
+use App\AutoMapping;
+use App\Controller\BaseController;
+use App\Request\Image\ImageCreateRequest;
+use App\Service\Image\ImageService;
+use stdClass;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Serializer\SerializerInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+use OpenApi\Annotations as OA;
+
+/**
+ * @Route("v1/imagetest/")
```

Comment:
this is helpful only for human tests, we do not wanna it. replace this controller with Command.

Reply:
this can not help us at all when doing PHP unit test for controller